### PR TITLE
Feat/metafields

### LINKED ISSUE #132: studio: Ability to add SEO Meta Content to Post
Similar to the Code Injection, Ghost provides a pretty clean interface to add SEO Meta Data: https://ghost.org/changelog/post-meta/

Provide a similar interface in Dega to add SEO Meta Data.

Pre-requisite for this issue: https://github.com/factly/dega-server/issues/271

### LINKED ISSUE #133: studio: Add `MetaFields` to all the entities
Add MetaFields property that we have on Category model to all the entities. This provides the ability to the users to add any content to the entity that is not part of the existing fields.

Closing this issue is a pre-requisite: https://github.com/factly/dega-server/issues/269

### PR DESCRIPTION
**Pull request by Bhumi101097**
Studio changes for meta fields and meta data
Closes #133 
Closes #132

### CODE REVIEW

File: studio/src/components/Template/index.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ function Template({ format }) {
   const { Panel } = Collapse;
   const page = 1;
   const [show, setShow] = React.useState(false);
-  const { posts, loading, media } = useSelector((state) => {
+  const { posts, loading } = useSelector((state) => {
```
Comment (shreeharsha-factly) [inline studio/src/components/Template/index.js:17]: Remove `media` on line 37 and 39

File: studio/src/pages/claimants/components/ClaimantForm.js [hunk 2]
```diff
@@ -35,13 +41,22 @@ const ClaimantForm = ({ onCreate, data = {} }) => {
     });
   };
 
+  const getJsonVal = (val) => {
+    let regex = /,(?!\s*?[{["'\w])/;
+    let formattedJson = val.replace(regex, '');
+    return JSON.parse(formattedJson);
+  };
+
```
Comment (shreeharsha-factly) [inline studio/src/pages/claimants/components/ClaimantForm.js]: Function `getJsonVal` looks repeated in all form components. Add that function in utils.

File: studio/src/pages/webhooks/components/WebhookForm.js [hunk 3]
```diff
@@ -33,7 +33,7 @@ const WebhookForm = ({ onCreate, data = {} }) => {
     page: 1,
     limit: 20,
   });
-  const { events, total, loading } = useSelector((state) => {
+  const { events } = useSelector((state) => {
     let details = [];
     let ids = [];
     let total = 0;
```
Comment (shreeharsha-factly) [inline studio/src/pages/webhooks/components/WebhookForm.js]: Remove `total` and `loading` where ever it is not used.

**Review by shreeharsha-factly — CHANGES_REQUESTED** [review]

**Comment by deshetti** [comment]
@Bhumi101097 Please add the test cases as well for the functionality

**Review by shreeharsha-factly — APPROVED** [review]

**Pull request merged**